Autopilot/Inc/SensorFusion.hpp

```diff
@@ -0,0 +1,62 @@
+/**
+ * Sensor Fusion - Converts raw sensor data into human readable formatted structs
+ * Author: Lucy Gong, Dhruv Rawat, Anthony Bertnyk
+ */
+
+#include "AttitudeDatatypes.hpp"
+#include "fetchSensorMeasurementsMode.hpp"
```

Comment:
What exactly are you including from "fetchSensorMeasurementsMode.hpp"? 

Asking because Chief will be working on a PR that will axe this module (since SF will be collecting data the fetchSensorMeasurements module is redundant)

Reply:
Right, I can remove the include now that SF will collect the data directly instead.